### PR TITLE
Backports it to guillotina 3.2.18

### PR DESCRIPTION
**Pull request by lferran**

### CODE REVIEW

**Comment by lferran** [comment]
@vangheem I'm getting 401 on the `test_api.py`. Seems like root does not have access to the package endpoints.

It's weird. Any idea why? Is `guillotina.Manage` not given to `root` by default?

File: setup.py [hunk 1]
```diff
@@ -36,7 +36,7 @@
             'pytest-aiohttp',
             'pytest-cov',
             'coverage>=4.4',
-            'pytest-docker-fixtures[rabbitmq]==1.2.5',
+            'pytest-docker-fixtures[rabbitmq]==1.2.7',
```
Comment (vangheem) [inline setup.py]: It's not a good idea to use hard-pins in setup.py--you're forcing consumers to use this.

Can you change it to `>=`?

**Review by vangheem — CHANGES_REQUESTED** [review]

File: setup.py [hunk 2]
```diff
@@ -9,14 +9,14 @@
 
 setup(
     name='guillotina_amqp',
-    version='2.0.4.dev0',
+    version='2.1.0.dev0',
     description='Integrate amqp into guillotina',
     long_description=README,
     install_requires=[
-        'guillotina>=4.0.0',
+        'guillotina==3.2.18',
```
Comment (vangheem) [inline setup.py]: Do not use fixed version here. `>=`.

File: setup.py [hunk 3]
```diff
@@ -9,14 +9,14 @@
...
         'aioamqp',
         'lru-dict',
-        'guillotina_rediscache>=2.0.4',
+        'guillotina_rediscache<=2.0.4',
```
Comment (vangheem) [inline setup.py]: Why this?

Comment (vangheem) [reply setup.py]: Just remove the version constraint if it works forward and backward.

**Review by vangheem — APPROVED** [review]

**Review by vangheem — CHANGES_REQUESTED** [review]
please fix version pinning issues.

**Comment by vangheem** [comment]
oh, I just noticed this isn't guillotinaweb. You forked it? I guess you can do whatever you want if you forked it then :)

As far as I can see, `guillotina.Manage` isn't actually a permission defined anywhere.

**Comment by lferran** [comment]
yes I forked it just so guillotina 3.2.18 can use it :+1: 
Thanks for the quick feedback !!

**Comment by lferran** [comment]
ouch.. the tests are passing green but actually they are failing:
```
INFO:guillotina_amqp.worker:Finished task: 80284853-9344-4d1c-adce-64d935249d7b: guillotina_amqp.utils._run_object_task
PASSEDDEBUG:guillotina_amqp:autokill: No beacons received
ERROR:guillotina_amqp:Exiting worker because of no beacon activity
The command "pytest -s --cov=guillotina_amqp -s --tb=native -v --cov-report term-missing --cov-append guillotina_amqp" exited with 0.
```

This might aswell be happening in the upstream branch

**Comment by vangheem** [comment]
See https://github.com/guillotinaweb/guillotina_amqp/commit/497bbc6ae97b1143dfa28c1027ca93490f5f886e ?

**Comment by lferran** [comment]
mmm it must be something else, since this commit is also included in this fork.
I'll look tomorrow

**Comment by lferran** [comment]
> `guillotina.AccessContent` is really open permission. I suggest creating your own permission and granting `guillotina.Manager` role.

@vangheem I'll fix this on the upstream branch aswell, ok? 
https://github.com/guillotinaweb/guillotina_amqp/pull/12

**Comment by lferran** [comment]
> ouch.. the tests are passing green but actually they are failing:
> 
> ```
> INFO:guillotina_amqp.worker:Finished task: 80284853-9344-4d1c-adce-64d935249d7b: guillotina_amqp.utils._run_object_task
> PASSEDDEBUG:guillotina_amqp:autokill: No beacons received
> ERROR:guillotina_amqp:Exiting worker because of no beacon activity
> The command "pytest -s --cov=guillotina_amqp -s --tb=native -v --cov-report term-missing --cov-append guillotina_amqp" exited with 0.
> ```
> This might aswell be happening in the upstream branch

yup, the same is happening in the main repo: https://travis-ci.org/guillotinaweb/guillotina_amqp/builds/470408125?utm_source=github_status&utm_medium=notification

**Pull request merged**